### PR TITLE
Update API URL from cloud.redhat.com to console.redhat.com

### PR DESCRIPTION
**Pull request by chambridge**
* Update items for API URL

### CODE REVIEW

File: api/v1beta1/metricsconfig_types.go [hunk 1]
```diff
@@ -20,10 +20,10 @@ const (
 	// CertCheck allows certificate validation to occur.
 	CertCheck bool = true
 
-	//UploadOn sets the operator to upload to cloud.redhat.com.
+	//UploadOn sets the operator to upload to console.redhat.com.
```
Comment (maskarb) [inline api/v1beta1/metricsconfig_types.go:23]:
```suggestion
	// UploadOn sets the operator to upload to console.redhat.com.
```

File: api/v1beta1/metricsconfig_types.go [hunk 2]
```diff
@@ -20,10 +20,10 @@ const (
...
 	UploadOn bool = true
 
-	//UploadOff sets the operator to not upload to cloud.redhat.com.
+	//UploadOff sets the operator to not upload to console.redhat.com.
```
Comment (maskarb) [inline api/v1beta1/metricsconfig_types.go:26]:
```suggestion
	// UploadOff sets the operator to not upload to console.redhat.com.
```

**Review by maskarb — APPROVED** [review]
I'm curious, should we have the `ReflectSpec` change the spec from `cloud` to `console`?

Otherwise, this looks good to me.

**Comment by chambridge** [comment]
> ReflectSpec

You mean making a change here:
https://github.com/project-koku/koku-metrics-operator/blob/main/controllers/kokumetricsconfig_controller.go#L115

To essentially swap out the "cloud.redhat.com" to "console.redhat.com" if their spec has that?

I was going to perhaps do that in a follow-on PR. Wanted to at minimum get this in code, but I can do that here if you prefer.

**Comment by maskarb** [comment]
> > ReflectSpec
> 
> You mean making a change here: https://github.com/project-koku/koku-metrics-operator/blob/main/controllers/kokumetricsconfig_controller.go#L115
> 
> To essentially swap out the "cloud.redhat.com" to "console.redhat.com" if their spec has that?
> 
> I was going to perhaps do that in a follow-on PR. Wanted to at minimum get this in code, but I can do that here if you prefer.

A follow up PR is totally cool with me!

**Review by maskarb — APPROVED** [review]

**Pull request merged**